DOCUMENTATION.md

```diff
@@ -4000,7 +4001,7 @@ const cfg = {
 
 ## 8.13. Configuring Load Balancer
 
-Load Balancer configuration allows you to specify which cluster member to send next operation when queried.
+Load Balancer configuration allows you to specify which cluster member to send the following operation when queried.
```

Comment:
the sentence is too complex. 

```suggestion
Load Balancer allows you to send operations to one of a number of endpoints (members).
```

from https://docs.hazelcast.com/hazelcast/5.2-snapshot/clients/java#configuring-client-load-balancer